### DISCUSSION TITLE
Inbucket Cloud Deployment

### DISCUSSION THREAD

**Discussion by jonstewart24** [inbucket/inbucket#455] [Q&A]
If you install Inbucket on AWS (or any cloud), do you just need an MX record on the domain pointed at the instance where Inbucket is accessible?

Only open required port traffic is the web client and SMTP port you have configured?

I've installed on AWS ECS with a network load balancer with port 25 listener and open in all security groups, and added an MX Record pointing at the NLB but no traffic is coming through.

**Comment by jhillyerd** [comment]
Yes, 25 should be the only port required to receive mail.  You do need to configure Inbucket to listen on port 25, as it defaults to 2500 (otherwise it would require root or privileged access on Unix systems).  I'd recommend trying to telnet/netcat to port 25 each stage in the chain to see where it breaks: localhost, hosts IP from the host itself, hosts IP from outside the host, load balancer, etc.
